packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -55,6 +56,15 @@ export class TriggerManager extends Component {
     }
   }
 
+  componentDidMount() {
+    try {
+      CozyFolder.registerClient(this.props.client)
+    } catch (error) {
+      if (error.message !== 'Document cannot be re-registered to a client.')
```

Comment:
This assumes than the client used does not change. It is dangerous IMHO to make this assumption.

Reply:
Should we have something similar to `cozy-authentication`? https://github.com/cozy/cozy-libs/blob/master/packages/cozy-authentication/src/MobileRouter.jsx#L84 

Cozy-client should expose the list of its events, and we should map on this list to update the client used by cozy-doctypes.

I don't think we currently have a use case where a client can be different in `Document`?